src/RedisCache/RedisCache/help/Set-AzRedisCache.md

```diff
@@ -100,7 +322,7 @@ Accept wildcard characters: False
 ```
 
 ### -Name
-Specifies the name of the Redis Cache to update.
+Specifies the name of the Azure Cache for Redis to update.
```

Comment:
Parameter description is generated from CSharp code. You should update below code 
https://github.com/Azure/azure-powershell/blob/cbba6dc1ff2ebe0bffb0b284022c7559102aafa1/src/RedisCache/RedisCache/Commands/SetAzureRedisCache.cs#L35 and generate docs again according to instruction https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-a-single-markdown-file

New-AzRedisCache.md has the similar issue.

Reply:
I have updated the help message in code and run the help generation commands.